<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [jbkQASuite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="QuizApp Test(failed)">
    <classes>
      <class name="com.jbkqa.testCases.TC_32_SignUpDataDrivenTest">
        <methods>
          <include name="signUpFailedScreenshot"/>
          <include name="SignUpDataDrivenTest" invocation-numbers="0 1 2 3 4 5 6 7"/>
          <include name="tearDown"/>
          <include name="setUp"/>
        </methods>
      </class> <!-- com.jbkqa.testCases.TC_32_SignUpDataDrivenTest -->
      <class name="com.jbkqa.testCases.TC_31_ProfileUpdateTest">
        <methods>
          <include name="tearDown"/>
          <include name="setUp"/>
          <include name="profileUpdate"/>
        </methods>
      </class> <!-- com.jbkqa.testCases.TC_31_ProfileUpdateTest -->
      <class name="com.jbkqa.testCases.TC_02_LoginDataDrivenTest">
        <methods>
          <include name="tearDown"/>
          <include name="setUp"/>
          <include name="LoginDataDrivenTest" invocation-numbers="1 3"/>
          <include name="loginFailedScreenshot"/>
        </methods>
      </class> <!-- com.jbkqa.testCases.TC_02_LoginDataDrivenTest -->
    </classes>
  </test> <!-- QuizApp Test(failed) -->
</suite> <!-- Failed suite [jbkQASuite] -->
